cli/src/terminal.rs

```diff
@@ -1,4 +1,20 @@
 use crate::errors::ErrorDetails;
+use console::Term;
+
+// For interactively handling user input
+pub fn input(msg: &str, sensitive: bool) -> Result<String, ErrorDetails> {
```

Comment:
Instead of a boolean for sensitive, I think this could be clearer with an enum for ReadMode

```rs
enum ReadMode {
  Secure
  Normal
}
```

Or something like that. Mainly to make the call sites more explict since `input` is likely to be used a lot!